requirements.txt

```diff
@@ -1 +1,2 @@
+flake8-isort==2.4
```

Comment:
Can you create a separate `requirements-test.txt` file with flake8-isort?
The requirements.txt lists packages needed to use the project, `requirements-test.txt` lists the packages needed to work on the project.

Reply:
Sure.